sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/BeamCalciteTable.java

```diff
@@ -67,11 +67,17 @@ public RelDataType getRowType(RelDataTypeFactory typeFactory) {
 
   @Override
   public Statistic getStatistic() {
-    BeamRowCountStatistics beamStatistics =
-        beamTable.getRowCount(BeamEnumerableConverter.createPipelineOptions(pipelineOptions));
-    return beamStatistics.isUnknown()
-        ? Statistics.UNKNOWN
-        : Statistics.of(beamStatistics.getRowCount().doubleValue(), ImmutableList.of());
+    final ClassLoader originalClassLoader = Thread.currentThread().getContextClassLoader();
+    try {
+      Thread.currentThread().setContextClassLoader(BeamEnumerableConverter.class.getClassLoader());
```

Comment:
Can you add a comment that points to `BeamEnumerableConverter` and explains that this is required for JDBC path?

Reply:
Sure. I'll add it.